purr/patterns/right-sidebar.php

```diff
@@ -3,6 +3,8 @@
  * Title: Right Sidebar
  * Slug: purr/right-sidebar
  * Categories: sidebar
```

Comment:
this is not one of the [default categories](https://developer.wordpress.org/themes/patterns/registering-patterns/#pattern-categories). Unless we had more than one pattern for it, we shouldn't create new categories

Reply:
We have 2 patterns for sidebar left-sidebar & right-sidebar

Left Sidebar has already sidebar category added, so I need to remove sidebar categories in left-sidebar.php pattern also?

Reply:
mmh, maybe in this theme it makes sense to have a sidebar category, but to have it show up in the pattern inserter we need to [register](https://developer.wordpress.org/themes/patterns/registering-patterns/#registering-a-pattern-category) it first if it's not one of the default ones

Reply:
Right, I will Register sidebar category first.

Reply:
I have created functions.php to register sidebar Pattern Category.

Reply:
![sidebar-pattern-category](https://github.com/WordPress/community-themes/assets/22027190/5f6b9031-49bd-4b83-82d1-69ba761650f0)